.travis.yml

```diff
@@ -17,3 +13,4 @@ before_script:
   - npm prune
 after_success:
   - npm run semantic-release
+  - npm run deploydocs
```

Comment:
We only want to deploy docs if a release succeeded. in `after_success`, all commands get run, even if a previous one failed, I've only learned this myself recently.

We could either do `- npm run semantic-release && npm run deploydocs`, or what I would suggest is to add `"postpublish": "deploydocs"` to the package.json in `"scripts"`.